src/lang/modifyAst/faces.spec.ts

```diff
@@ -182,7 +182,12 @@ extrude001 = extrude(profile001, length = 10, tagEnd = $capEnd001)
       expect(newCode).toContain(
         `shell001 = shell(extrude001, faces = END, thickness = 1)`
       )
-      await enginelessExecutor(ast, undefined, undefined, rustContextInThisFile)
+      await enginelessExecutor(
+        result.modifiedAst,
+        undefined,
+        undefined,
+        rustContextInThisFile
+      )
```

Comment:
This is just cleanup throughout the file: those test cases weren't checking the new ast for mock exec errors! Thanks bot